C-Plus-Plus/graphs/FloodFillAlgorithm/FloodFill(usingDFS).cpp

```diff
@@ -0,0 +1,90 @@
+/*Flood Fill Algorithm determines the area connected to given node in multidensioanl array
+
```

Comment:
You may add both the algorithms in the same code file using Object-Oriented Design, Hint - Creating two classes for BFS and DFS respectively and objects of the same for the algorithms.

Reply:
@Abhijit2505 I have made a single file by including both functions in same program. Please review it again.